tests/tinitcommand.nim

```diff
@@ -0,0 +1,17 @@
+import unittest, os
+import testscommon
+
+from nimblepkg/common import cd
+
+suite "init":
+  ## https://github.com/nim-lang/nimble/pull/983
+  test "init within directory that is invalid package name will not create new directory":
+    cleanDir(installDir)
+    let tempdir = installDir / "a-b"
+    createDir(tempdir)
```

Comment:
I think you want to use getTempDir here.

Reply:
Fixed. How do I pass CI tests?

Reply:
Looks like you've got warnings in your changes: https://github.com/nim-lang/nimble/runs/5265139850?check_suite_focus=true#step:6:221